src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -0,0 +1,56 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+import lombok.*;
```

Comment:
the wildcard import approach has advantages, but its usage can cause problems in certain situations

please turn off Wildcard imports (https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-)